library/core/networking/src/main/java/org/quiltmc/qsl/networking/api/ServerPlayNetworking.java

```diff
@@ -60,6 +80,7 @@ public final class ServerPlayNetworking {
 	 * @see ServerPlayNetworking#unregisterGlobalReceiver(Identifier)
 	 * @see ServerPlayNetworking#registerReceiver(ServerPlayNetworkHandler, Identifier, ChannelReceiver)
 	 */
+	@Deprecated
```

Comment:
still not a fan of these `@Deprecated` methods, but I'll leave them alone;
just make sure to have `@deprecated` notes on the javadoc

Reply:
Yeah im still not sure what i want to do with these. I originally had these marked for removal but i didnt like that.